manga_translator/manga_translator.py

```diff
@@ -184,7 +197,11 @@ async def translate(self, image: Image.Image, params: dict = None) -> Context:
             else:
                 ctx.alignment = 'auto'
         ctx.setdefault('renderer', 'manga2eng' if ctx['manga2eng'] else 'default')
-        if ctx.translator_chain is not None:
+        if ctx.selective_translation is not None:
+            ctx.selective_translation.target = ctx.target_lang
+            ctx.selective_translation.default = ctx.selective_translation.langs[0]
```

Comment:
1. Remove default property, make dispatch use first language as default on its own.
2. target variable naming is ambiguous. Rename to target_lang